server/migrations/20160212102110-add-avg-to-snippet.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var models = require('../models');
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    queryInterface.addColumn(
+      'Snippets',
+      'avg',
+      {
+        type: Sequelize.FLOAT,
+        allowNull: false,
+        defaultValue: 0.0
+      }
+    );
+    models.Snippet.findAll()
+    .then(function(snippets) {
+      snippets.map(function(snippet) {
+        models.Rating.aggregate('value', 'avg', {where: {SnippetId: snippet.id}})
```

Comment:
@blimusiek didn't we agreed that this part should go to task instead of migration?